src/main/scala/amba/axi4/Fragmenter.scala

```diff
@@ -31,7 +30,7 @@ class AXI4Fragmenter()(implicit p: Parameters) extends LazyModule
       val beatBytes = slave.beatBytes
       val lgBytes   = log2Ceil(beatBytes)
       val master    = edgeIn.master
-      val masters   = master.masters
+      master.masters
```

Comment:
Hmm, should this line just be deleted entirely?

Reply:
I don't understand much this "optimization". But it seems like that Scalafix removed an assignment to the local "masters" similar to the constant propagation for a Unit value.
The same name in the local scope with not functional modifications.

Actually, each "val" assumes a memory alloc on JVM, so removing extra "vals" saves on allocs